src/Console.h

```diff
@@ -0,0 +1,24 @@
+#ifndef CONSOLE_H_
+#define CONSOLE_H_
+
+#include <iostream>
+
+
+class Console
+{
+public:
+    Console() = default;
+    ~Console() = default;
+
+    static bool nextCommand(const std::string& command);
+    static bool pathIsCorrect(const std::string& path);
+
+    static void createDB();
+    static bool uploadDB(const std::string& path);
+
+private:
+
```

Comment:
There is no required hash map, use map<string, function<void(vector<string params>)>> to declare such

Reply:
> There is no required hash map, use map<string, function<void(vector)>> to declare such

Added